### PR TITLE
UI redesign using new components

### PR DESCRIPTION
**Pull request by iamoperand**

### CODE REVIEW

File: packages/frontend/src/components/Charts/ShortFundingChart.tsx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+      gap: '5px',
+    },
+    iconWrapper: {
+      display: 'flex',
+      marginLeft: 6,
+    },
+  }),
+)
+
+const FundingChart = () => {
```
Comment (alexisgauba) [inline packages/frontend/src/components/Charts/ShortFundingChart.tsx:39]: Notice that the title says short funding chart, but we should show it both long and short when we add back the tabs, to be consistent with data on the current site

Comment (alexisgauba) [reply packages/frontend/src/components/Charts/ShortFundingChart.tsx:39]: not sure where the line color is, but we should change it to #70E3F6 for consistency with the new blue in the new designs

**Review by alexisgauba — COMMENTED** [review]
Ticket has this weird behavior on scroll 
https://user-images.githubusercontent.com/13340486/203431403-8f09d007-b46a-4d5c-bd5b-d0c54650501c.mov

Don't need the warning for "collateralization ratio is risky" since it already says risky on the bar w no new additional info. The 150% warning is good because it provides additional info. Also the 150% warning is more of  a real issue, so don't want "collateralization ratio is risky" to dilute from that 
Screen Shot 2022-11-22 at 1 23 07 PM
Screen Shot 2022-11-22 at 1 23 16 PM

On screens that aren't even that small (I'm on a 13in Macbook Air) the ticket stacks - would be good to have it on the right in there situations 
https://user-images.githubusercontent.com/13340486/203431641-ad24fb54-202c-4594-bcd3-e76d9e1f1660.mov

**Comment by alexisgauba** [comment]
There's a few screen sizes where we get this overlap before it goes from desktop to mobile 


https://user-images.githubusercontent.com/13340486/203693643-9715ea03-1660-4f15-916f-4b53110266bd.mov

**Comment by alexisgauba** [comment]
There's some screen size before it goes from desktop view to mobile view where the graph overlaps with the trade ticket 


https://user-images.githubusercontent.com/13340486/203693893-ca5c0591-18d0-410a-b0d6-03ace9ac6bf2.mov

**Comment by alexisgauba** [comment]
13in full screen stacks 😅 

Screen Shot 2022-11-24 at 11 54 43 AM

**Comment by alexisgauba** [comment]
We should also change the colors on the long / short tabs to red & green, but the new red and green

**Comment by alexisgauba** [comment]
Todos from user test: 

- [ ] Make long & short tabs red & green 
	- User did get confused about when was on long vs. short
- [ ] Change line colors on graph -> squeeth PnL to squeeth blue, and ETH to purple 
- [ ] Add background to actions on the right (let’s wait on zk for this) 
	- Comes from user feedback for 3 main UI elements (left side info, toggle that affects whole page from long / short, and action on right), and not clear delineation between each 
- [ ] Issues with responsiveness (was stacked, and then refreshed to unstack, seems like the position arrow thing causing the box to get bigger / smaller could be impacting here) 
- [ ] On the long side the data didn’t load for him when he inputted a value [add ss] 
- [ ] Make the buy button bigger (button a little thin, should be chunkier) 
- [ ] On the short ticket, move the “sell” section above the “collateral” section (wait for zk on this) 
- [ ] On the crab page make the strategy hedges a scrollable item where it shows 3 and then you scroll in that card to see the rest

**Comment by alexisgauba** [comment]
Gettin this error when trying to buy long squeeth 

Also price impact isn't loading on the long side 

Screen Shot 2022-11-24 at 10 44 53 PM

**Comment by alexisgauba** [comment]
Todos from internal testing: 

- [ ] Change “Long Squeeth - ETH² Position” to “Long Squeeth - ETH² Leverage”
- [ ] Cannot type “.0” and add a digit after .0 (for example .05 ETH). You need to type .5 and then add 0 in front of the 5 to arrive at .05
ezgif com-gif-maker - 2022-11-25T170835 552

- [ ] Change "earnings" to "PnL" on crab page 
- [ ] Remove "deposited value" on crab page
- [ ] Center the confirmation screen
Screen Shot 2022-11-25 at 3 20 59 PM

**Comment by alexisgauba** [comment]
Porting over the todo to fix the details alignment

Screen Shot 2022-11-25 at 3 28 39 PM


To fix, could align the top row with the others, or also attaching the screenshot from the designs as another possibilty
Screen Shot 2022-11-25 at 3 29 10 PM

**Comment by reganomalley** [comment]
The graph is being set to the full width of the screen only when you resize the window. If you reload at any screen width, it’s fine. 
Screenshot 2022-11-28 at 3 04 04 PM

**Comment by reganomalley** [comment]
When you go to open a long position, it's no
Screenshot 2022-11-28 at 3 13 33 PM
t fetching the oSQTH price relative to the inputted ETH amount. 

See screenshot.

**Comment by reganomalley** [comment]
Projections not working 
Screenshot 2022-11-28 at 3 21 37 PM

**Comment by reganomalley** [comment]
My first mainnet txn failed. My second worked. Could be a fluke, but wanted to mark it: 


https://etherscan.io/tx/0x7987b0768e6f64fc93c6410241e6d0518a04611433f0979aac8daec0a9f91b02

**Comment by reganomalley** [comment]
Says I bought 0 oSQTH, which is wrong 
Screenshot 2022-11-28 at 3 30 15 PM

**Comment by alexisgauba** [comment]
zk list of prios before launch 

- [ ] Responsiveness issues (graph collision) 
Screenshot 2022-11-28 at 3 04 04 PM

- [ ] Change the colors on long/short to the new red/green 

- [ ] Change line colors in graph -> squeeth PnL to blue and ETH to purple 

- [ ] Add background to actions on the right 

	- See the the "To Implement" Frame on the "ideas for action ticket" page on the Figma, which adds a background with a dropshadow for the ticket
	Screen Shot 2022-11-27 at 12 37 14 PM

- [ ] On the long side the data didn’t load when inputted a value 

Screen Shot 2022-11-24 at 1 31 16 PM

- [ ] Make the buy button bigger (button a little thin, should be chonkier) 

- [ ] Gettin this error when trying to buy long squeeth 

Screen Shot 2022-11-24 at 10 44 53 PM

- [ ] Price impact not loading on the long side (see above)

- [ ] Change “Long Squeeth - ETH² Position” to “Long Squeeth - ETH² Leverage”

- [ ] Cannot type “.0” and add a digit after .0 (for example .05 ETH). You need to type .5 and then add 0 in front of the 5 to arrive at .05

ezgif com-gif-maker - 2022-11-25T170835 552

- [ ] Change "earnings" to "PnL" on crab page 
- [ ] Remove "deposited value" on crab page
- [ ] Center the confirmation screen
Screen Shot 2022-11-25 at 3 20 59 PM

- [ ] Add tooltips back to the "details" boxes 
- [ ] Tooltip on the short side for the initial approval before closing a position, shows up under the button 
Screen Shot 2022-11-27 at 2 09 19 PM

- [ ] Fix the sizing of the Short PnL box 
Screen Shot 2022-11-27 at 2 07 48 PM

- [ ] Move "manage vault" to "manage" on the position value card on the short side
Screen Shot 2022-11-28 at 7 47 54 PM

- [ ] Projections not working 
Screenshot 2022-11-28 at 3 21 37 PM

- [ ] Says bought 0 oSQTH, which is wrong 

Screenshot 2022-11-28 at 3 30 15 PM

**Comment by alexisgauba** [comment]
Review comments 

- [ ] Not letting me input anything if my wallet not connected

https://user-images.githubusercontent.com/13340486/204896346-fdc1316c-9946-4c94-a63e-3fd9f01b0db5.mov

**Review by alexisgauba — APPROVED** [review]

**Pull request merged**